File: Programmers/java/LEVEL_2_JadenCase_문자열_만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";

        StringBuilder sb = new StringBuilder(s);

        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) >= '0' && s.charAt(i) <= '9') {
                continue;
            }

            if (i == 0 || s.charAt(i - 1) == ' ') {
                String str = String.valueOf(s.charAt(i));
                str = str.toUpperCase();
                char c = str.charAt(0);
                sb.setCharAt(i, c);
                continue;
            }

            String str = String.valueOf(s.charAt(i));
            str = str.toLowerCase();
            char c = str.charAt(0);
            sb.setCharAt(i, c);
        }

        answer = sb.toString();

        return answer;
    }
}

File: Programmers/cpp/LEVEL_2_JadenCase_문자열_만들기.cpp
#include <string>
#include <vector>
#include <cctype>

using namespace std;

string solution(string s) {
	string answer = "";

	for (int i = 0; i < s.length(); i++) {
		if (s[i] >= '0' && s[i] <= '9') {
			continue;
		}

		if (i == 0 || s[i - 1] == ' ') {
			s[i] = toupper(s[i]);
			continue;
		}

		s[i] = tolower(s[i]);
	}

	answer = s;

	return answer;
}